### PR TITLE
Feature/kuz468 memory storage controller

### PR DESCRIPTION
**Pull request by benoitvidis**
* New _memoryStorage_ controller, aliased to _ms_
* All redis commands are exposed, **excepted**:
  * _script_ related commands
  * _cluster_ related commands
  * _pubsub_ commands
  * _cursor_ commands
* ResponseObject now accepts a result that can be resolved to false

### CODE REVIEW

File: lib/api/controllers/routerController.js [hunk 1]
```diff
@@ -142,68 +143,207 @@ module.exports = function RouterController(kuzzle) {
...
-        {verb: 'put', url: '/users/:id', controller: 'security', action: 'createOrReplaceUser'},
-        {verb: 'put', url: '/:index', controller: 'admin', action: 'createIndex'},
-        {verb: 'put', url: '/:index/:collection', controller: 'write', action: 'createCollection'},
-        {verb: 'put', url: '/:index/:collection/_mapping', controller: 'admin', action: 'updateMapping'},
-        {verb: 'put', url: '/:index/:collection/:id/_:action', controller: 'write'},
-        {verb: 'put', url: '/:index/:collection/:id', controller: 'write', action: 'createOrReplace'}
-      ];
+      coverage;
+
+    routes = [
```
Comment (scottinet) [inline lib/api/controllers/routerController.js]: I'm starting to believe that we should move these routes into a configuration file :-)

**Comment by scottinet** [comment]
Tremendous work!

+1

**Comment by AnthonySendra** [comment]
+1

**Comment by j33f** [comment]
+1

**Pull request merged**